app/controllers/publishers_controller.rb

```diff
@@ -26,6 +26,18 @@ class PublishersController < ApplicationController
   before_action :require_verified_email,
     only: %i(email_verified
              complete_signup)
+  before_action :require_publisher_email_not_verified_through_youtube_auth,
```

Comment:
minor: as we add new actions we might forget to whitelist them here.  maybe we could use `:except` here?

Reply:
Good idea